src/utils/coordinates.js

```diff
@@ -1,5 +1,5 @@
 /* global THREE */
-
+var extend = require('object-assign');
```

Comment:
I'm doing something similar in another PR. A `cloneIfObject` util would be helpful